CHANGELOG.md

```diff
@@ -23,7 +24,31 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [v1.140.0] - 2023-11-17
 
-* Fix EOC bands in `nursing-internal` 
+* Fix EOC bands in `nursing-internal`
+* Style `boxed-feature` note in `neuroscience`
+* Change `suggested-reading` to one column in `nursing-external`
+* Style EOC sections in `neuroscience`
+* Apply `FullWidthTable` to `nursing-external`
+* Style `appendix` and `index` in `neuroscience`
+
+* Style `appendix references` in `lifespan-development`
+* Style third level lists in `carnival`
+* Fix GitPod support
+* Style `answer key` in `lifespan-development`
+* Style `notes` in `lifespan-development`
+* Style `appendix glossary` in `lifespan-development`
+* Style `index` in `lifespan-development`
+* Style `eoc` sections in `lifespan-development`
+* Style `tables` in `lifespan-development`
+* Style `lo` in `lifespan-development`
+* Style `chapter-outline` in `lifespan-development`
+* Style figures in `lifespan-development`
+* Style headers in `lifespan-development`
+* Create basic files and style TOC in `lifespan-development`
+
+## [v1.140.0] - 2023-11-17
+
+* Fix EOC bands in `nursing-internal`
```

Comment:
I think conflicts were resolved in a wrong way. There are repetitions in a CHANGELOG. Would you mind to compare this list of logs with one from main branch to make sure that versions which were already tagged are not changed and that nothing is missing from `Unreleased` one?